src/test/integration/nodemailer.test.ts

```diff
@@ -0,0 +1,65 @@
+import enmail from '../../index';
+
+jest.mock('nodemailer');
+const nodemailer = require('nodemailer');
+nodemailer.createTransport.mockReturnValue({ sendMail: jest.fn(data => data) });
+
+describe('Nodemailer adapter', () => {
+    const getGmailAdapter = () => enmail.createAdapter.nodemailer(options);
+    const options = {
+        service: 'gmail',
+        auth: {
+            type: 'login' as any,
+            user: 'test2@example.com',
+            pass: '********',
+        },
+    };
+    const adapter = getGmailAdapter();
+    const sendOptions = { bcc: 'test3@example.com' };
+    const body = {
+        to: 'test@example.com',
+        from: 'test2@example.com',
+        content: 'Hello world',
+        subject: 'Hello',
+    };
+    it('Gmail adapter can be created', () => {
+        expect(typeof adapter.sender).toEqual('function');
+        expect(adapter.options).toStrictEqual(options);
+        expect(typeof adapter.mailService.sendMail).toEqual('function');
+        expect(adapter).toMatchSnapshot();
+    });
+    it('Sender will be successfully returned', () => {
+        const sender = enmail.sender(adapter);
+        expect(typeof sender).toEqual('function');
+        expect(sender).toEqual(adapter.sender);
+    });
+    it('An email can be sent via adapter', async () => {
+        const gmailMessage = await adapter.sender()(body, sendOptions) as any;
+        expect(Object.keys(gmailMessage).sort()).toEqual(['bcc', 'to', 'html', 'from', 'subject'].sort());
```

Comment:
This expect is repeated 7 times without a change. Should be refactored into a function.

Reply:
i created `expect` function